addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -320,6 +383,37 @@ const LinkTools = Link.extend({
             this.options.wysiwyg.odooEditor.historyStep();
         }
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onUploadFileClick(ev) {
+        const documentDialog = new MediaDialog(this, {noIcons: true, noVideos: true, noImages: true});
+        documentDialog.on('save', this, data => {
+            if (data) {
+                const {src, href, title} = data;
+                let url = href || src;
+                url = url.replace(/[&?]+download=true/, '').replace(window.location.origin, '');
```

Comment:
I'd add a comment here, you can't resolve this comment if you think this is not needed/is obvious without the comment:
something like
// download=true automatically added by dialog save in js, we don't want it, default file link should open in need tab (and will be configurable in link tool)

Reply:
done